packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -32,7 +35,52 @@ const Color _kDefaultGlowColor = Color(0xFFFFFFFF);
 @immutable
 class ScrollBehavior {
   /// Creates a description of how [Scrollable] widgets should behave.
-  const ScrollBehavior();
+  const ScrollBehavior({
+    @Deprecated(
+      'Temporary migration flag, do not use. '
+      'This feature was deprecated after v2.1.0-11.0.pre.'
+    )
+    bool useDecoration = false,
+    this.scrollbarPlatforms,
+    this.glowingPlatforms,
```

Comment:
instead of using `glowing`, maybe this should be called something a little more generic, like overscrollIndicator?